dimod/constrained.py

```diff
@@ -1780,10 +1780,11 @@ def substitute_self_loops(self) -> Dict[Variable, Variable]:
             >>> cqm = ConstrainedQuadraticModel()
             >>> cqm.add_constraint(i*i <=3, label='i squared')
             'i squared'
-            >>> cqm.substitute_self_loops()                      # doctest: +IGNORE_RESULT
-            >>> cqm.constraints   # doctest: +IGNORE_RESULT
-            {'i squared': QuadraticModel({'i': 0.0, 'cf651f3d-bdf8-4735-9139-eee0a32e217f': 0.0}, {('cf651f3d-bdf8-4735-9139-eee0a32e217f', 'i'): 1.0}, 0.0, {'i': 'INTEGER', 'cf651f3d-bdf8-4735-9139-eee0a32e217f': 'INTEGER'}, dtype='float64') <= 3,
-            'cf651f3d-bdf8-4735-9139-eee0a32e217f': QuadraticModel({'i': 1.0, 'cf651f3d-bdf8-4735-9139-eee0a32e217f': -1.0}, {}, 0.0, {'i': 'INTEGER', 'cf651f3d-bdf8-4735-9139-eee0a32e217f': 'INTEGER'}, dtype='float64') == 0}
+            >>> cqm.substitute_self_loops() #doctest: +ELLIPSIS
+            {...}
```

Comment:
This is nice, thank you, but can you say that the ellipses are a stand in for the generated var name?